documents/kratos_doxyfile

```diff
@@ -2248,7 +2248,7 @@ PREDEFINED             = KRATOS_API(...)=
 # definition found in the source code.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-EXPAND_AS_DEFINED      =
+EXPAND_AS_DEFINED      = KRATOS_EXPRESSION_UTILS_CEXP_METHOD_1 KRATOS_EXPRESSION_UTILS_CEXP_METHOD_2 KRATOS_EXPRESSION_UTILS_CEXP_METHOD_3
```

Comment:
I don't think this will be a sustainable solution, but I guess it'll have to do for now. That said, please break it into multiple lines at least:

```suggestion
EXPAND_AS_DEFINED      = KRATOS_EXPRESSION_UTILS_CEXP_METHOD_1 \
                         KRATOS_EXPRESSION_UTILS_CEXP_METHOD_2 \
                         KRATOS_EXPRESSION_UTILS_CEXP_METHOD_3
```